js/background.js

```diff
@@ -0,0 +1,17 @@
+const githubUrlRegex = /https?:\/\/github\.com\/.*\/network\/members/;
+if (typeof browser === "undefined") {
+    var browser = chrome;
+}
+
+browser.tabs.onUpdated.addListener(function (tabId, info, tab) {
+	if (
+		info.status === "complete" &&
+		tab.title.indexOf("Fork") !== -1 &&
```

Comment:
Is this check really necessary? I don't like it because it depends on the language of the browser.
GitHub is only available in English, but I've seen people use the translator and the latter also changes the title, if I'm not mistaken.

Reply:
Thanks for the review. I have the same opinion as you.  But if I delete this code, it will not work properly.  

GitHub has called `browser.tabs.onUpdated.addListener` several times, two of those times `info.status === "complete"`. I add `tab.title.indexOf("Fork") !== -1` for apply the process only once at the end.   

If you know of another workaround, it would be better to go that way.

Reply:
Ironically, I found that this code does not work in this repository.  Because this repository name is "fork"-diff.  
I convert this pull request to draft. This process may need to be radically changed.  I'm sorry.

Reply:
Don't worry, thank you very much for the PR!

Reply:
Thank you. If you find a solution, please ignore this pull request and fix it.